Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM node:8.15-alpine
+
+WORKDIR /app
+
+RUN apk add --update \
+    bash \
+    git \
+    rsync \
+    zip \
+    python3 \
```

Comment:
What do we need python3 for? o_O

Reply:
They are used during testing, see https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-s3-deployment/lambda/test.sh#L19-L22